src/persist-types/src/stats.rs

```diff
@@ -1050,112 +1047,118 @@ mod impls {
         }
     }
 
-    impl StatsFrom<Bitmap> for PrimitiveStats<bool> {
-        fn stats_from(col: &Bitmap, validity: ValidityRef) -> Self {
-            let array = BooleanArray::new(
-                arrow2::datatypes::DataType::Boolean,
-                col.clone(),
-                validity.0.as_ref().cloned(),
-            );
-            let lower = arrow2::compute::aggregate::min_boolean(&array).unwrap_or_default();
-            let upper = arrow2::compute::aggregate::max_boolean(&array).unwrap_or_default();
+    impl StatsFrom<BooleanBuffer> for PrimitiveStats<bool> {
+        fn stats_from(col: &BooleanBuffer, validity: ValidityRef) -> Self {
```

Comment:
Happy to leave this for a follow-up, but should we update all these "validity" names to match the `arrow` jargon, which seems to be something like "logical nulls"?

Reply:
That sounds good to me! I'll do that in a follow-up